src/dvsim/launcher/base.py

```diff
@@ -90,6 +90,48 @@ class Launcher(ABC):
         context=[],
     )
 
+    def __init__(self, deploy: "Deploy") -> None:
```

Comment:
Do we need the quotes here? I'd have thought that it only matters when we're type checking, in which case it's in scope?

Reply:
I agree with both of those things. My question was why the quotes are needed in this particular case.

But I think I've realised: even if the type annotation doesn't do anything at runtime, all of its symbols still need to be in scope.